www/front_src/src/Header/userMenu/index.tsx

```diff
@@ -131,53 +112,53 @@ const useStyles = makeStyles((theme) => ({
     width: theme.spacing(0),
   },
   icon: {
-    minWidth: theme.spacing(3.75),
+    marginRight: theme.spacing(1),
   },
   icons: {
-    alignItems: 'center',
     borderLeft: `1px solid ${theme.palette.common.white}`,
-    display: 'flex',
-    gap: theme.spacing(2),
-    [theme.breakpoints.down(1200)]: {
-      gap: theme.spacing(1),
-      paddingLeft: theme.spacing(2.5),
+    paddingLeft: theme.spacing(3),
+  },
+  listItem: {
+    '&:first-child': {
+      borderBottom: `1px solid ${theme.palette.grey[300]}`,
     },
-    paddingLeft: theme.spacing(4),
-    [theme.breakpoints.down(900)]: {
-      paddingLeft: theme.spacing(1.5),
+    '&:hover': {
+      background: theme.palette.primary.light,
+      color: theme.palette.primary.main,
     },
-    [theme.breakpoints.down(640)]: {
-      borderLeft: 'none',
+    '&:last-child': {
+      borderTop: `1px solid ${theme.palette.grey[300]}`,
     },
+    padding: theme.spacing(1),
+  },
+  listItemButton: {
+    '&:hover': {
+      background: 'none',
+    },
+    padding: 0,
   },
   menu: {
-    backgroundColor: equals(theme.palette.mode, ThemeMode.dark)
-      ? theme.palette.background.default
-      : theme.palette.primary.main,
+    // backgroundColor: equals(theme.palette.mode, ThemeMode.dark)
+    //   ? theme.palette.background.default
+    //   : theme.palette.primary.main,
+    backgroundColor: theme.palette.background.paper,
     border: 'none',
     borderRadius: 0,
-    color: theme.palette.common.white,
+    fontSize: '.75rem',
```

Comment:
Is it possible to use a fontSize from a MUI typography (body1, body2, caption, etc...) ?